polar-core/src/error.rs

```diff
@@ -344,6 +349,29 @@ The expression is: {expr}
                     expr = term.to_polar(),
                 )
             }
+            Self::DataFilteringFieldMissing { var_type, field } => {
+                let msg = formatdoc!(
+                    r#"Unregistered field or relation: {}.{}
+                    
+                    Please include `{}` in the `fields` parameter of your
+                    `register_class` call for {}.  For example, in Python:
+
+                        oso.register_class({}, fields={{
+                            {:?}: <type or relation>
+                        }})
+
+                    For more information please refer to our documentation:
+                        https://docs.osohq.com/guides/data_filtering.html
+                    "#,
+                    var_type,
+                    field,
+                    field,
+                    var_type,
+                    var_type,
+                    field
```

Comment:
it'd be easier to match all these variables w/ the corresponding format directive if we used the named syntax, e.g., `"{var_type}", var_type=var_type` instead of `"{}", var_type`

Reply:
Ooooh cool I did not know you could do this, thanks!